index.test.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-env jest */
 // @ts-check
-import "./index.js";
-import { add } from "./math/index.js";
+import std from "./index.js";
 
-add(5, 3);
+test("the module be defined", () => {
```

Comment:
`the module is defined`

Reply:
This test is now skipped and will be removed in the future. Thanks for pointing this out.